### PR TITLE
Java Github actions for the tutorial example

### PR DESCRIPTION
**Pull request by Artem-applitools**

### CODE REVIEW

**Comment by Artem-applitools** [comment]
secrets `GITHUB_TOKEN` was changed to the `TOKEN` in comparison to the other tutorials repos as I was unable to create secret with such name.

**Comment by Artem-applitools** [comment]
Also I have updated the `eyes.setApiKey()` to use env variable `APPLITOOLS_API_KEY`
And set the chrome to headless if the java system property `ci_test` is `true`

File: src/test/java/com/applitools/quickstarts/BasicDemo.java [hunk 1]
```diff
@@ -10,15 +10,17 @@
...
 public class BasicDemo {
 
 	@Test
         public void test() {
 
-		// Use Chrome browser
-		WebDriver driver = new ChromeDriver();
 
+		// Use Chrome browser
+		boolean CI_TEST = Boolean.getBoolean("ci_test");
```
Comment (colbyfayock) [inline src/test/java/com/applitools/quickstarts/BasicDemo.java:22]: in JavaScript, we're able to get `process.env.CI` in GitHub Actions which would equal `true`. are we able to use that instead? not sure if we're able to access it the same. it should be available as an environment variable if so. just less code for someone to worry about to utilize what's already in the Action

**Comment by Artem-applitools** [comment]
Oh, it's possible in java as well, but requere small code update. Will
update the code tomorrow.

вт, 18 мая 2021 г., 19:11 Colby Fayock ***@***.***>:

File: src/test/java/com/applitools/quickstarts/BasicDemo.java (hunk 1 above)
Comment (Artem-applitools) [reply src/test/java/com/applitools/quickstarts/BasicDemo.java]: Updated

**Comment by colbyfayock** [comment]
awesome 🙌

File: .github/workflows/updates.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        run: mvn versions:use-latest-releases
+
+
+      - run: mvn clean -Dtest=BasicDemo test
+        env:
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+
+      - uses: mikeal/publish-to-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```
Comment (colbyfayock) [inline .github/workflows/updates.yml:31]: hey just so you know for future reference, this should be ${{ secrets.GITHUB_TOKEN }}

GitHub will automatically set that value from the repository but you still need to explicitly pass it in

i fixed this on the main branch

**Pull request merged**